zcnbridge/keystore.go

```diff
@@ -39,7 +39,7 @@ func AccountExists(address string) bool {
 		return false
 	}
 
-	status, err := wallet.Status()
+	status, _ := wallet.Status()
```

Comment:
do you think it should return false if err is not nil?

Reply:
I am not sure how it is defined. Wallet can have any status and be thought as existed.
Even the call to `wallet.Status()` is done to just print the information.